Silksong.ModMenu.csproj

```diff
@@ -51,23 +50,12 @@
     <PackageReference Include="Silksong.GameLibs" Version="*-*" PrivateAssets="all" />
   </ItemGroup>
   <ItemGroup Condition="$(UseLocalRefs) And Exists($(ManagedFolder))">
-    <Reference Include="Assembly-CSharp">
-      <HintPath>$(ManagedFolder)\Assembly-CSharp.dll</HintPath>
-    </Reference>
-    <Reference Include="Assembly-CSharp-firstpass">
-      <HintPath>$(ManagedFolder)\Assembly-CSharp-firstpass.dll</HintPath>
-    </Reference>
-    <Reference Include="TeamCherry.Localization">
-      <HintPath>$(ManagedFolder)\TeamCherry.Localization.dll</HintPath>
-    </Reference>
-    <Reference Include="UnityEngine.UI">
-      <HintPath>$(ManagedFolder)\UnityEngine.UI.dll</HintPath>
-    </Reference>
+    <LocalReferences Include="$(ManagedFolder)*.dll" Exclude="$(ManagedFolder)Mono*.dll;$(ManagedFolder)netstandard.dll;$(ManagedFolder)System*.dll;$(ManagedFolder)mscorlib.dll" />
+    <Reference Include="@(LocalReferences)" Private="false" />
   </ItemGroup>
   <ItemGroup>
     <Folder Remove="thunderstore/tmp;thunderstore/dist" />
     <None Include="README.md" Pack="True" PackagePath="README.md" />
-    <None Include="thunderstore/icon.png" Pack="True" PackagePath="icon.png" />
```

Comment:
You will need this for the icon on nuget to work (in fact I suspect the build will fail if you `dotnet pack`)

Reply:
Blah, I guess VS removed this when I replaced the icon.  This wasn't manual